src/guides/v2.3/rest/operation-status-endpoints.md

```diff
@@ -51,6 +51,17 @@ Field name | Data type | Description
 }
 ```
 
+## Get operations count by bulk uuid and status.
+
+The `GET <host>/rest/<store_code>/V1/bulk/:bulkUuid/status/:status` endpoint returns amount of operations from the bulk batch in particular status.
```

Comment:
Please, make sure that you are mentioning the correct endpoint here. It looks like it supposed to be `/V1/bulk/:bulkUuid/operation-status/:status` but I might be wrong. 

Thank you.